hoomd/extern/jitify.hpp

```diff
@@ -688,7 +690,7 @@ inline bool load_source(
     source += line + "\n";
   }
   // HACK TESTING (WAR for cub)
-  // source = "#define cudaDeviceSynchronize() cudaSuccess\n" + source;
+  source = "#define cudaDeviceSynchronize() cudaSuccess\n" + source;
```

Comment:
I wonder if this change is necessary. I'm not sure how it relates to getting more verbose errors.